website/docs/cloud-docs/api-docs/state-versions.mdx

```diff
@@ -48,12 +48,16 @@ Attribute                        | Description
 ---------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 `hosted-json-state-download-url` | A URL from which you can download the state data in a [stable format](/terraform/internals/json-format) appropriate for external integrations to consume. Only available if the state was created by Terraform 1.3+.
 `hosted-state-download-url`      | A URL from which you can download the raw state data, in the format used internally by Terraform.
+`hosted-json-state-upload-url`   | A URL to which you can upload the state data, in a [stable format](/terraform/internals/json-format) appropriate for external integrations to consume. Only available if the JSON state content has not been uploaded.
+`hosted-state-upload-url`        | A URL to which you can upload the state data, in the format used internally by Terraform. Only available if the state content has not been uploaded.
 `modules`                        | Extracted information about the Terraform modules in this state data. Populated asynchronously.
 `providers`                      | Extracted information about the Terraform providers used for resources in this state data. Populated asynchronously.
+`provisional`                    | A boolean flag that denotes a state version that was created but has not yet marked as the current version for a workspace.
```

Comment:
should this be state "version", or state "snapshot"? i worry state version is a little loaded since we have that field in the state file also

Reply:
We are avoiding using the term "snapshot", because a snapshot is indistinguishable from a state version. The key difference is that a "snapshot" is a state version that has not been set as the current version for the workspace. Rather than introduce a different term altogether, we decided to use the term "provisional" state version.